client/gameManager.js

```diff
@@ -415,6 +424,7 @@ export class GameManager {
       input: this.input,
       assets: next.assets,
       libs: this.libs,
+      gameState: this.state,
```

Comment:
can we consider a different name from `gameState` here? it may be a bit confusing next to `this.state` that represents the current game's state. maybe `meta` or `mirco`?